<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="custom_plot" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>571</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>572</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdd_Point">
    <property name="title">
     <string>Add Point</string>
    </property>
    <addaction name="actionIndependent"/>
    <addaction name="actionOn_line"/>
    <addaction name="actionOn_Circle"/>
    <addaction name="actionMidpoint"/>
   </widget>
   <widget class="QMenu" name="menuAdd_Line">
    <property name="title">
     <string>Add Line</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAdd_Circle">
    <property name="title">
     <string>Add Circle</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuRemove">
    <property name="title">
     <string>Remove</string>
    </property>
    <addaction name="actionRemove"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionEdit"/>
   </widget>
   <widget class="QMenu" name="menuAdd_Triangle">
    <property name="title">
     <string>Add Triangle</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTriangle_Centers">
    <property name="title">
     <string>Triangle Centers</string>
    </property>
   </widget>
   <addaction name="menuAdd_Point"/>
   <addaction name="menuAdd_Line"/>
   <addaction name="menuAdd_Circle"/>
   <addaction name="menuAdd_Triangle"/>
   <addaction name="menuTriangle_Centers"/>
   <addaction name="menuEdit"/>
   <addaction name="menuRemove"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionIndependent">
   <property name="text">
    <string>Independent</string>
   </property>
  </action>
  <action name="actionMidpoint">
   <property name="text">
    <string>Midpoint</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove Construction</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit Point</string>
   </property>
  </action>
  <action name="actionOn_line">
   <property name="text">
    <string>On line</string>
   </property>
  </action>
  <action name="actionOn_Circle">
   <property name="text">
    <string>On Circle</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
